docusaurus/docs/reference/deployments/01-controller.md

```diff
@@ -6,24 +6,126 @@ sidebar_label: Controller
 
 import CliLogFormats from '../../_cli-log-levels-and-formats.md'
 
-This article provides some guidance for deploying a Ziti Controller. You can go back and read [the controller component introduction](/learn/introduction/03-components.md#openziti-controller) if needed.
+This article is about deploying a Ziti Controller as a Linux system service. [The controller introduction](/learn/introduction/03-components.md#openziti-controller) introduces the architectural concepts used in this guide. [Kubernetes is covered separately](/guides/kubernetes/hosting/kubernetes-controller.mdx).
 
-## Public Key Infrastructure
+## Installation
 
-You will need a PKI setup for Ziti. If you follow one of the [quickstart guides](/learn/quickstarts/network/hosted.md) then a PKI will be generated for you. This is probably the best way to start out. Pay particular attention to the PKI-related fields in the configuration file, and check out [the pki page](/learn/core-concepts/pki.md).
+The controller package is installed the same way as the `ziti` CLI and depends on the `openziti` package the provides the CLI.

```

Comment:
This is the right place to put that installaction stuff from above. I'd remove most of the words here and just show them how to install it